jcl/src/java.management/share/classes/java/lang/management/PlatformLoggingMXBean.java

```diff
@@ -37,7 +37,7 @@
  * </li>
  * <li>Using a javax.management.MBeanServerConnection.</li>
  * <li>Obtaining a proxy MXBean from the static
- * {@link ManagementFactory#newPlatformMXBeanProxy} method, passing in
+ * {@link java.lang.management.ManagementFactory#newPlatformMXBeanProxy} method, passing in
```

Comment:
The link on line 36 needs to be fixed as well:
` {@link java.lang.management.ManagementFactory#getPlatformMXBean}`